strategyexplorer.ui

```diff
@@ -6,8 +6,8 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>1037</width>
-    <height>721</height>
```

Comment:
A little bit worried here, it might cause 720p devices unable view the full panel of strategy explorer.

Reply:
I see. The height is not as important as the width: it was the smaller width, that lead to the left table not being fully visible, but having a scroll bar added. I increased the height just so that the field would be more square, but as said, that part is not as important.

Reply:
Okay, I see, thanks for clearification~